File: CursoPOOUber/Java/PayPal.java
package Java;

public class PayPal extends Payment{
    Integer id;
    float monto;
    String fecha;
    String email;
    
    public PayPal( float monto, String fecha, String email){
        super(monto, fecha, email);

        this.monto = monto;
        this.fecha = fecha;
        this.email = email;
    }
}

File: CursoPOOUber/Python/paypal.py
from payment import Payment

class Paypal:

    id = int
    monto = float
    fecha = str
    email = str

    def __init__(self, monto, fecha, email):
        super().__init__(monto, fecha, email)
        self.monto = monto
        self.fecha = fecha
        self.email = email